loadtest/scripts/populate_genesis_accounts.py

```diff
@@ -34,45 +43,106 @@ def add_genesis_account(account_name, lock, local=False):
             "mnemonic": mnemonic,
         }
         json.dump(data, f)
-    success = False
-    retry_counter = 5
+
+    return add_account_cmd
+
+
+def create_genesis_account(account_index, account_name, local=False):
+    address, mnemonic = add_key(account_name=account_name, local=local)
+    add_account_cmd = add_account(account_name=account_name, address=address, mnemonic=mnemonic, local=local)
+
+    retry_counter = 0
     sleep_time = 1
-    while not success and retry_counter > 0:
+
+    while True:
         try:
-            with lock:
-                subprocess.check_call(
-                    [add_account_cmd],
-                    shell=True,
-                    timeout=20,
-                )
-                success = True
+            print(f'Running: ${add_account_cmd}')
+            subprocess.call(
```

Comment:
We're essentially leaving the mutex to Seid command, it already does a bunch of checks against the genesis state when writing to it. 

This script will spam it until it passes

Reply:
for safety, should we set a max of like 1k and if it still fails then error out?